Dockerfile.ppc64le

```diff
@@ -1,17 +1,27 @@
 # Build the manager binary
```

Comment:
Please remove this file. Since the Dockerfile is parameterized for multi-arch builds.